docs/draw_table.js

```diff
@@ -87,8 +86,8 @@ function renderTable(list, features, selectTableCallback, selectorReceiver) {
 
   google.visualization.events.addListener(
       table, 'select',
-      () => selectTableCallback(
-          table.getSelection().map((elt) => features[elt.row])));
+      (features) => highlightFeatures(
```

Comment:
is features passed to highlight features no longer the features param that's being passed in to renderTable?

Reply:
Great catch! integration_tests/highlight_features_test failed with this bug, so I guess we have some coverage :)